test/syscalls/vector/Makefile

```diff
@@ -14,7 +14,7 @@
 
 EXAMPLE=vector
 #CC=riscv64-unknown-linux-gnu-gcc
-CC="${RVCC}"
+CC="${RVCXX}"
```

Comment:
Why are you setting `CC` to `"${RVCXX}"` instead of `"${RVCC}"`?

Reply:
ah that was a quick fix while I was testing. good catch